test/e2e/policies.test-suite/topology-aware/n4c16/test00-basic-placement/nri-resmgr.cfg.in

```diff
@@ -5,6 +5,8 @@ policy:
   topology-aware:
     ColocatePods: $(echo ${COLOCATE_PODS:-false})
     ColocateNamespaces: $(echo ${COLOCATE_NAMESPACES:-false})
+instrumentation:
+  HTTPEndpoint: ":8891"
```

Comment:
the reason I am adding them here is that we are using the same daemonSet for all the TA tests and I've added the liveness probe there. If we don't enable instrumentation in those TA tests then liveness probe fail so the test too.

Reply:
I agree, I think we should have a dedicated test case for the liveness probe / health check which then will need the endpoint enabled, but only for that test case.

Reply:
@jukkar @klihub sorry for the confusion. Let me clarify it a bit why I had to add `HTTPEndpoint` in other TA tests config files.

We have a single [nri-resmgr-topology-aware-deployment.yaml.in](https://github.com/jukkar/nri-resmgr/blob/main/test/e2e/files/nri-resmgr-topology-aware-deployment.yaml.in) that is used for all the TA plugin tests. We don't have/generate unique deployment files (daemonSet object) for each test case even though we have different configuration file for them. It is true that e2e is structured in a way that if you pass a custom configuration file to the test case under the directory `testxx-abc/` then that particular test will use that instead of default one (test/e2e/policies.test-suite/topology-aware/nri-resmgr.cfg).

I have added a separate test case for the health-checker `test12-health-checking`, which requires enabling HTTP endpoint on the configuration file it is using + readiness probe on the deployment file. This test case can have it is own configuration file but it is not needed. Because, the idea of the test is - let's launch the nri-rm with the default configuration file (doesn't matter if the HTTP endpoint is given or not) and then create a configMap with HTTP enabled + some syntactically incorrect data which should case the readiness probe to fail. 

If readiness probe is enabled on the daemonSet file but nri-rm is not exposing HTTP endpoint, then the container will fail (not considered as running). Since a single deployment file is used across all the tests of TA, and since I added readiness probe to the same file, I had to add HTTP endpoint to the custom configuration file of other tests and to the default one. Otherwise, we end up with a daemonSet with readiness probe enabled while nri-rm not exposing the HTTP endpoint and that will cause the failure for other tests.

I could add a separate daemonSet YAML for only health-checker test, but we don't usually create custom daemonSet and as said it is only configuration file that might change.